src/react/lwt_react.cppo.ml

```diff
@@ -5,13 +5,7 @@
 
 open Lwt.Infix
 
-let opaque_identity x =
-  #if OCAML_VERSION < (4, 03, 0)
-    x
-  #else
-    Sys.opaque_identity x
-  #endif
-
+let opaque_identity x = Sys.opaque_identity x
```

Comment:
maybe just remove this binding entirely, since below you're using `Sys.opaque_identity` explicitly anyways ;)

Reply:
It is exported in the API. It is exported after a `(**/**)` so probably not a big issue to remove it. Still we might want to add a deprecation annotation.

It is also used in one of the tests but we could use the fully qualified version too.

Reply:
Resolved from #949